### PR TITLE
BOOKKEEPER-1001 Make LocalBookiesRegistry.isLocalBookie() public

### PR DESCRIPTION
**Pull request by caiok**

### CODE REVIEW

**Comment by eolivelli** [comment]
LGTM +1

**Comment by jvrao** [comment]
+1 LGTM

File: bookkeeper-server/src/test/java/org/apache/bookkeeper/test/LocalBookiesRegistryTest.java [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+                .newServerConfiguration()
+                .setDisableServerSocketBind(true)
+                .setEnableLocalTransport(true);
+    }
+        
+    BookKeeper.DigestType digestType;
+    
+    public LocalBookiesRegistryTest(BookKeeper.DigestType digestType) {
+        super(3);
+        this.digestType=digestType;
```
Comment (sijie) [inline bookkeeper-server/src/test/java/org/apache/bookkeeper/test/LocalBookiesRegistryTest.java]: space before and after '='?

Comment (caiok) [inline bookkeeper-server/src/test/java/org/apache/bookkeeper/test/LocalBookiesRegistryTest.java]: Corrected, thanks

**Pull request closed without merging**